docs/python/inference/conf.py

```diff
@@ -50,14 +56,44 @@
 exclude_patterns = []
 pygments_style = "default"
 autoclass_content = "both"
+master_doc = "index"
+onnx_doc_folder = os.path.join(os.path.abspath(os.path.dirname(__file__)), "operators")
+pygments_style = "sphinx"
 
 # -- Options for HTML output -------------------------------------------------
 
-html_theme = "alabaster"
+html_static_path = ["_static"]
+html_theme = "pydata_sphinx_theme"
```

Comment:
This will change the look and feel of all of the python API docs right? (Which is not bad) But to stage this before publishing, can you:
- merge these changes into the main of your fork
- this will generate a PR to the gh-pages branch of your fork
- merge this and add a link to the GitHub pages site generated from your fork
Let me know if you need any help to do this

Reply:
Yes it will change the style. I used the same style as onnx. I'll publish the pages to show how it looks.

Reply:
This is how it looks: https://xadupre.github.io/draft/inference/index.html.